eclipse-scout-core/src/form/fields/sequencebox/SequenceBox.js

```diff
@@ -193,6 +196,11 @@ export default class SequenceBox extends CompositeField {
     }
   }
 
+  _onLastVisibleFieldSuppressStatusChanged(e) {
```

Comment:
According to our naming convention it should be called XyChange instead of XyChanged